projects/plugins/jetpack/extensions/blocks/blogroll/utils.js

```diff
@@ -5,21 +5,10 @@ export function createBlockFromRecommendation( attrs ) {
 
 	return createBlock( 'jetpack/blogroll-item', {
 		...attrs,
-		icon: getSiteIconOrPlaceholder( icon ),
+		icon: icon || 'https://s0.wp.com/i/webclip.png',
```

Comment:
No longer needed.

Reply:
We should keep this to display a placeholder image in the editor.
If I remove this and the site doesn't have a icon, the image is broken in the editor.